Sources/Flatten.swift

```diff
@@ -164,6 +181,9 @@ extension Signal where Value: SignalProducerConvertible, Error == NoError, Value
 
 		case .race:
 			return self.race()
+
+		case .once:
+			return self.take(first: 1).race()
```

Comment:
On the description you explain it's equivalent to `flatMap(.latest)`, but internally you are using `race`.

Reply:
Since there's only one signal, all the strategies are functionally equivalent. The only difference is the performance characteristics.

Reply:
We do not have a case for performance here yet, just the room for optimization.

> Since there's only one signal, all the strategies are functionally equivalent. 

`race` propagates `interrupted` from the inner stream to the flattened stream. The other strategies do not.

Reply:
Hmm, so the strategy is relevant.